src/components/StaffPayment/StaffPayment.vue

```diff
@@ -1,15 +1,15 @@
 <template>
-  <v-card flat id="staff-payment-container" class="mt-4 py-8 pr-6 rounded-0">
+  <v-card flat id="staff-payment-container" class="py-8 px-6">
```

Comment:
Again, I'm thinking this shouldn't be a v-card. The parent can wrap this in a v-card if it needs to.

Reply:
I changed it.